api/src/main/java/org/openmrs/module/covid19/COVID19Activator.java

```diff
@@ -48,6 +51,8 @@
 	
 	private static final Logger log = LoggerFactory.getLogger(COVID19Activator.class);
 	
+	public static final String INSTALLED_VERSION_GP = "covid19.installedVersion";
```

Comment:
Do we do anything with this?

Reply:
Yeah sure i havent used this , i have removed it